src/main/java/com/google/cloud/genomics/utils/Contig.java

```diff
@@ -94,12 +94,12 @@ public Contig apply(String contigString) {
           }
         });
   }
-  
+
   // The following methods have package scope and are helpers for ShardUtils. For sharded Contigs,
   // the ShardUtils methods should be used to ensure that shards are shuffled all together before
   // being returned to clients.
   List<Contig> getShards(long numberOfBasesPerShard) {
-    double shardCount = Math.ceil(end - start) / (double) numberOfBasesPerShard;
+    double shardCount = Math.ceil(end - start) / numberOfBasesPerShard;
```

Comment:
Thanks :)  BTW, I'm sure you noticed this as well, but `shardCount` can be `int` if you cast `Math.ceil` as follows, since you use it as an integer upper-bound in the for-loop afterwards:

```Java
int shardCount = (int) Math.ceil( (end - start) / numberOfBasesPerShard );
```